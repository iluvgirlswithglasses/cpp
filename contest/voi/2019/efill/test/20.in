37
-4058242 3338332
-4058212 -3625300
3274492 -4058228
-4040466 4052012
-2684174 -4058220
3894256 -4056308
4058162 2178840
-4057990 3965444
-4058210 3500970
3974414 4056922
4051326 4042844
4058158 -3574366
4056420 -3927820
4058088 -3675292
-4054416 -3996002
-4055690 4035708
4049468 4051992
3803522 4058010
4053148 -4013544
-705610 4058242
4058230 -103286
-3863198 4056850
-3799892 4057640
4057096 3901548
4018220 -4047280
-1170710 -4058230
-4057260 -3812800
-4039432 -4058120
4004496 -4054232
3777676 -4057124
-4045074 -4040850
4054158 3977720
4058230 1323220
3420032 4058088
4057654 3680984
4058208 -2023000
-3300498 4058186
-87621811 -143844 -31912755 -143604
