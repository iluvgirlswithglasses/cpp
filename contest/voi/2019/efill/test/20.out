110567466.27
